RELEASES.md

```diff
@@ -1,3 +1,194 @@
+Version 1.53.0 (2021-06-17)
+============================
+
+Language
+-----------------------
+- [You can now use unicode for identifiers.][83799] This allows multilingual
+  identifiers but still doesn't allow glyphs that not considered characters
+  such as `◆` or `🦀`. More specifically you can now use any identifier that
+  matches the UAX #31 "Unicode Identifier and Pattern Syntax" standard. This
+  is the same standard as languages like Python, however Rust uses NFC
+  normalisation which may be different from other languages.
+- [You can now specify "or patterns" inside pattern matches.][79278]
+  Previously you could only use `|` (OR) on complete patterns. E.g.
+  ```rust
+  let x = Some(2u8);
+  // Before
+  matches!(x, Some(1) | Some(2));
+  // Now
+  matches!(x, Some(1 | 2));
+  ```
+- [Added the `:pat_param` `macro_rules!` matcher.][83386] This matcher
+  has the same semantics as the `:pat` matcher. This is to allow `:pat`
+  to change semantics to being a pattern fragment in a future edition.
+
+Compiler
+-----------------------
+- [Updated to build with LLVM 12 by , and enabled `mutable-noalias` when
+  built LLVM 12 or greater.][82834]
```

Comment:
Leaving a note that it's not clear yet whether we'll rever this or not.

Let's leave this comment unresolved and check back close to the release on what the status of https://github.com/rust-lang/rust/issues/84958 is.

Reply:
We've pretty much agreed to revert noalias, but the upgrade itself I thought was in the previous release?

Reply:
@Mark-Simulacrum where is the discussion on reverting noalias? I'm a bit surprised there's no trace of it in #84958.

Reply:
Yeah, the last thing on the thread seems to indicate a backport, not a revert, as there a decision not to?

Reply:
https://zulip-archive.rust-lang.org/131828tcompiler/20159shouldwedisablemutablenoaliasin153currentbeta.html

Still needs to be reflected into the thread, I was planning to do so eventually.